packages/caip/src/constants.ts

```diff
@@ -49,6 +49,7 @@ export const cosmosAssetId: AssetId = 'cosmos:cosmoshub-4/slip44:118'
 export const thorchainAssetId: AssetId = 'cosmos:thorchain-1/slip44:931'
 export const tcyAssetId: AssetId = 'cosmos:thorchain-1/slip44:tcy'
 export const mayachainAssetId: AssetId = 'cosmos:mayachain-mainnet-v1/slip44:931'
+export const mayaTokenAssetId: AssetId = 'cosmos:mayachain-mainnet-v1/slip44:maya'
```

Comment:
@kaladinlight not sure about the naming here, this seems like it may come and bite us later with human errors - probably worth backing off from our regular naming and make `mayachainAssetId` `cacaoAssetId` ?

Reply:
bleh... yeah this one sucks because `cacaoChainId` doesn't make sense and then I could see accidentally using `mayaTokenAssetId` in conjunction with `mayachainChainId`.

mayachain _is_ the chain, so I feel like even though close `mayachainAssetId` and `mayaAssetId` makes the most sense. Existing patterns are followed and it does mean what is says, they just happen to be named similarily

Reply:
That compromise sounds reasonable to me!

Reply:
2cee8fefe5 errego!